share/ansible_navigator/utils/image_introspect.py

```diff
@@ -115,6 +115,11 @@ def start_workers(self, jobs):
 
 
 class CmdParser:
+    @staticmethod
+    def _strip(value: str) -> str:
```

Comment:
This is a pure function, it shouldn't belong to a class.

Reply:
It's use is limited and specific to the class in which it is contained, and not intended to be used outside this class nor is it.

Reply:
It's just that it's an antipattern in Python. It comes from Java where you cannot do things differently. Plus the name `_strip()` is not semantically connected to this class.

Imagine you write a unit test for this method (which you probably should). You'll call `CmdParser._strip('   as\'d"f ')` — now this looks like this has something to do with the parser or the command.

This is why it's recommended to at least put this helper outside of the class.